utils/tests/utils.go

```diff
@@ -240,6 +240,17 @@ func GetBuildInfo(serverDetails *config.ServerDetails, buildName, buildNumber st
 	return servicesManager.GetBuildInfo(params)
 }
 
+func GetBuildRuns(serverDetails *config.ServerDetails, buildName, buildNumber string) (pbi *buildinfo.BuildRuns, found bool, err error) {
```

Comment:
This API ignore the buildNumber, see my comment on the `jfrog-cli-artifactory` PR.